src/main/webapp/modal-tester.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>TRACE</title>
+    <link rel="stylesheet" href="modal-stylesheet.css">
+    <script src="modal-activator.js"></script>
+  </head>
+  <body>
+    <Button> Load Results </Button>
+    <div class="modal" id="modal" name="results-popup">
+      <div class="modal-header"> 
+        <div class="modal-title"> Results </div>
+        <button class="exit-button"> &times;</button>
+      <div class="modal-body">
+        Lorem ipsum dolor sit amet, Nessun dorma! Nessun dorma! Tu pure, oh Principessa. 
+        Nella tua fredda stanza Guardi le stelle che tremano. D'amore e di speranza.
+    </div>
```

Comment:
not seeing where you closed modal-header and modal-body

Reply:
Addressed: 1f6d7e7